cmd/sync/aws.go

```diff
@@ -29,6 +30,24 @@ func NewAWSClient(data []byte) (*AWSClient, error) {
 		return nil, fmt.Errorf("error validating config: %v", err)
 	}
 
+	if cfg.Region == "self" {
+		metaSession, err := session.NewSession()
```

Comment:
I suggest using the cfg with the same timeout value as in the https://github.com/nginxinc/nginx-asg-sync/blob/ae94a73cefcec28990c283904f6f13502ad15e36/cmd/sync/aws.go#L79

Reply:
good suggestion. I didn't stick with the use of `cfg` due to it being consistently used in reference to the yaml config. I opted to use `params` instead.